BetterHolidays/markets/nasdaq.py

```
@@ -0,0 +1,160 @@
+            "nsdq_table"
+        )[0]
+        holidays = {}
+        for tr in table.body[0]:
+            name, day, status = (
+                tr.children[0].text,
+                tr.children[1].text,
+                tr.children[2].text,
+            )
+
+            month, date = day.split(" ")
+            date = dt.date(year, MONTHS_MAP[month.upper()], int(date))
```

Ruff (0.8.2)

Undefined name `year`

(F821)